browser/brave_wallet/brave_wallet_service_unittest.cc

```diff
@@ -168,6 +168,23 @@ class BraveWalletServiceUnitTest : public testing::Test {
     base::RunLoop().RunUntilIdle();
   }
 
+  void SetDefaultWallet(mojom::DefaultWallet default_wallet) {
+    service_->SetDefaultWallet(default_wallet);
+  }
+
+  mojom::DefaultWallet GetDefaultWallet() {
+    mojom::DefaultWallet default_wallet;
+    bool callback_called = false;
+    service_->GetDefaultWallet(
+        base::BindLambdaForTesting([&](mojom::DefaultWallet v) {
+          callback_called = true;
+          default_wallet = v;
+        }));
+    base::RunLoop().RunUntilIdle();
```

Comment:
why not `Run()` and just `Quit()` inside lambda? much better.

Reply:
Fixed here 7f0b2f0ffd0c79655515438bd5b6dddeccd59968